docs/tools/cli-ref-environment-variables.md

```diff
@@ -13,6 +13,9 @@ The behavior of the nuget.exe CLI can be configured through a number of environm
 
 In general, options specified directly on the command line or in NuGet configuration files have precedence, but there are a few exceptions such as *FORCE_NUGET_EXE_INTERACTIVE*. If you find that nuget.exe behaves differently between different computers, an environment variable could be the cause. For example, Azure Web Apps Kudu (used during deployment) has *NUGET_XMLDOC_MODE* set to *skip* to speed up package restore performance and save disk space.
 
+The NuGet CLI uses MSBuild to read the project files. All environment variables are available as [properties](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-command-line-reference) during the MSBuild evaluation.
```

Comment:
This isn't the correct syntax for linking docs from other repos: https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-other-repos-or-docsets

In particular, if someone's already browsing the docs in a language other than en-us, this link would take them to docs in the wrong language.